<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowDetail</class>
 <widget class="QWidget" name="ShowDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(119, 119, 179, 150);
</string>
  </property>
  <widget class="QPushButton" name="menu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
border: 2px;</string>
   </property>
   <property name="text">
    <string>메뉴로 돌아가기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
border: 2px;</string>
   </property>
   <property name="text">
    <string>다음</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,150);
font: 57 11pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
border: 2px;</string>
   </property>
   <property name="text">
    <string>이전</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(100, 100, 160, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ccontent">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>140</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 10pt &quot;DX별과그대&quot;;

background-color: rgb(255, 255, 255,15);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="viewpoint">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);
font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
border: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(100, 100, 160, 0);
font: 57 9pt &quot;DX별과그대B&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;추억 열어본 횟수&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대B&quot;;
background-color: rgba(100, 100, 160, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ffffff;&quot;&gt;추억 이름&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);
font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(206, 213, 255,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대B&quot;;
background-color: rgba(100, 100, 160, 0);
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#ffffff;&quot;&gt;추억 날짜&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cdate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);

font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대B&quot;;
background-color: rgba(119, 119, 179, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;추억 이벤트&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대B&quot;;
background-color: rgba(119, 119, 179, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;추억 장소&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>106</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 11pt &quot;DX별과그대B&quot;;

background-color: rgba(119, 119, 179, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;함께한 사람들&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpeople">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>177</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);
font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cplace">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>177</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);
font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cevent">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>177</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,15);
font: 57 9pt &quot;DX별과그대&quot;;
border-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menu</sender>
   <signal>clicked()</signal>
   <receiver>ShowDetail</receiver>
   <slot>open_main()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back</sender>
   <signal>clicked()</signal>
   <receiver>ShowDetail</receiver>
   <slot>Get_Back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>next</sender>
   <signal>clicked()</signal>
   <receiver>ShowDetail</receiver>
   <slot>Get_Next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_main()</slot>
  <slot>Get_Back()</slot>
  <slot>Get_Next()</slot>
 </slots>
</ui>
